feature-libs/user/account/components/verification-token-form/verification-token-form.component.html

```diff
@@ -28,7 +35,21 @@
     {{ 'verificationTokenForm.tokenInputHint' | cxTranslate }}
   </span>
 
-  <input type="hidden" formControlName="tokenId" [(ngModel)]="tokenId" />
+  <input
+    type="hidden"
+    formControlName="tokenId"
+    name="username"
+    [(ngModel)]="tokenId"
+  />
+
+  <ng-container *cxFeature="'authorizationCodeFlowByDefault'">
```

Comment:
do we have to wrap the <input>?
```suggestion
<input 
  *cxFeature="'authorizationCodeFlowByDefault'"
  *ngIf="csrf$ | async as csrf"
  formControlName="csrf"
  type="hidden"
  [name]="csrf.parameterName"
/>
```